luthier/src/test/groovy/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialParkSpec.groovy

```diff
@@ -1,6 +1,6 @@
 // Copyright 2019 Oath Inc.
 // Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
-package com.yahoo.bard.webservice.config.luthier
+package com.yahoo.bard.webservice.data.config.luthier;
```

Comment:
You fixed the package, but not the file location (should be under `webservice/data/config/luthier`).